### PR TITLE
NEWRELIC-3118: Node.js Agent ESM support

### PR DESCRIPTION
**Pull request by jmartin4563**
Update from @zuluecho9 Sept 26: added jira to review this in NR-52283. 

**What problems does this PR solve?**
The Node.js agent now supports instrumenting Node.js applications written using `import` and `export`, and has some specific setup steps that must be done. Additionally, updated the Node.js EOL matrix for versions that were missing 

**Add any context that will help us review your changes such as testing notes, links to related docs, screenshots, etc.**
Seeing as this is a brand new feature of the Node.js agent and a brand new page, would love to pair up with someone to ensure this page adheres to our voice/writing standards!

**If your issue relates to an existing GitHub issue, please link to it.**
Closes NEWRELIC-3118

### CODE REVIEW

**Review by bizob2828 — COMMENTED** [review]
my only comment would be finding a spot to link this in https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/

**Comment by michaelgoin** [comment]
@jmartin4563 I might recommend also updating the Compatibility and Requirements document to call out support and maybe link to the new document. (does not need to be this PR)

File: src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx [hunk 1]
```diff
@@ -363,6 +363,8 @@ For other message queue libraries, use [custom instrumentation](/docs/agents/nod
     title="Supported languages and technologies"
 >
 [TypeScript](https://www.npmjs.com/package/typescript) is a programming language that can compile to JavaScript. After installation, the Node.js agent automatically instruments your TypeScript apps, giving you immediate access to performance data right out of the box. 
+
+[ES Modules](https://nodejs.org/api/esm.html#introduction) are the official standard for packaging JavaScript code for sharing/reuse. The Node.js agent has **experimental** support for ES Module applications running Node.js version 16.12.0 or higher and agent version 9.1.0 or higher. Please see our [ES Module documentation](/docs/apm/agents/nodejs-agent/getting-started/es-modules) for additional information and instructions.
```
Comment (bizob2828) [inline src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx:367]: 👍🏻

**Review by bizob2828 — DISMISSED** [review]

**Comment by bizob2828** [comment]
@zuluecho9 any idea when you'll be able to review this?

File: src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx:367]: This looks good!

**Comment by rhetoric101** [comment]
@jmartin4563, this looks really good! Thanks for adding this new doc! 

I pushed up some minor tweaks. Could you look at those (the Gatsby preview is still running, but when that's done, you could look at the changes there or look right in the DIFF view? 

Also, what do you think about moving this over to the installation and configuration section?

**Review by bizob2828 — APPROVED** [review]

**Review by rhetoric101 — APPROVED** [review]
@jmartin4563 thanks again for all your work on these docs!

**Pull request merged**